Update README.md (#2)
amazon-braket/.github: profile/README.md

# Welcome to the Amazon Braket GitHub organization

> [Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry setting up cloud infrastructure. 

Reviewer: ```suggestion
[Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry about setting up cloud infrastructure. 
```

Quantum computing has the potential to solve computational problems beyond the reach of classical computers because it harnesses the laws of quantum mechanics to process information in new ways. 

> A number of open-source projects have been built ontop of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 

Reviewer: ```suggestion
A number of open-source projects have been built ontop of Amazon Braket established. On this page you will find starting points to discover these artifacts and how you can contribute to our quantum computing community! 
```

> A number of open-source projects have been built ontop of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 

Reviewer 2: ```suggestion
A number of open-source projects have been built on top of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 
```

> A number of open-source projects have been built on top of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 

Reviewer: ```suggestion
A number of open-source projects have been built on top of Amazon Braket. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 
```

# Amazon Braket fully supported repositories

> We aim to respond within few days for PRs and Issues raised in fully supported GitHub repositories. Customers can find these repositories installed on Amazon Braket Notebook Instances or on AWS managed containers for [Amazon Braket Hybrid Jobs](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-use.html).

Reviewer: ```suggestion
We aim to respond within a few days for PRs and Issues raised in fully supported GitHub repositories. Customers can find these repositories installed on Amazon Braket Notebook Instances or on AWS managed containers for [Amazon Braket Hybrid Jobs](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-use.html).
```

> * [Braket Python SDK](https://github.com/amazon-braket/amazon-braket-sdk-python) - Is a coding interface for quantum computing devices on Amazon Braket

Reviewer: ```suggestion
* [Braket Python SDK](https://github.com/amazon-braket/amazon-braket-sdk-python) - Provides a coding interface for quantum computing devices on Amazon Braket
```
nit: Keeps us from starting with a linking verb here

Author: Good call. Accepting change

* [Braket Examples](https://github.com/amazon-braket/amazon-braket-examples) - Contains tutorials on quantum computing with Amazon Braket
* [PennyLane-Braket Plugin](https://github.com/amazon-braket/amazon-braket-pennylane-plugin-python) - Enables PennyLane code to run on Amazon Braket
* [Qiskit-Braket Plugin](https://github.com/qiskit-community/qiskit-braket-provider) - Enables Qiskit code to run on Amazon Braket
* [Braket Algorithm Library](https://github.com/amazon-braket/amazon-braket-algorithm-library/tree/main) - Contains community and first party implementations of quantum algorithms

> * [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers/tree/main) - Offers a set of managed docker containers for hybrid jobs and good starting point for building custom containers

Reviewer: ```suggestion
* [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers/tree/main) - Offers a set of managed docker containers for hybrid jobs and a starting point for building custom containers
```

> * [Braket Default Simulators](https://github.com/amazon-braket/amazon-braket-default-simulator-python) - provides an implementation of quantum simulators that you can run locally

Reviewer: ```suggestion
* [Braket Default Simulators](https://github.com/amazon-braket/amazon-braket-default-simulator-python) - Provides an implementation of quantum simulators that can run locally
```

> * [Braket Schemas](https://github.com/amazon-braket/amazon-braket-schemas-python) - Used to construct and verify payloads being sent and generated by Amazon Braket

Reviewer: nit: verb tense differs from the rest of the descriptions. Fine to keep but also could word:
```suggestion
* [Braket Schemas](https://github.com/amazon-braket/amazon-braket-schemas-python) - Constructs and verifies payloads being sent and generated by Amazon Braket
```
